boards/BMS/Inc/App/App_Accumulator.h

```diff
@@ -8,6 +8,21 @@
 #include "App_CanTx.h"
 #include "App_TractiveSystem.h"
 
+// Accumulator Characteristics
+typedef enum
+{
+    ACCUMULATOR_SEGMENT_0 = 0U,
+    ACCUMULATOR_SEGMENT_1,
+    ACCUMULATOR_SEGMENT_2,
+    ACCUMULATOR_SEGMENT_3,
+    ACCUMULATOR_SEGMENT_4,
+    ACCUMULATOR_SEGMENT_5,
+    ACCUMULATOR_NUM_OF_SEGMENTS,
+} AccumulatorSegments_E;
```

Comment:
We should really clarify our conventions. Whether or not we typedef, whether or not we do _E/_T, etc.
I don't care for this PR because there's a big mix in our repo rn, but probably best we make conventions and stick to them (maybe at start of Jan?)